package.json

```diff
@@ -13,7 +13,7 @@
     "clean": "rm -rf node_modules/ dist/ tmp/",
     "crowdin": "bash ./tasks/download-translations.sh",
     "dev": "ember serve",
-    "fastboot-debug": "ember build && node --inspect --debug-brk ./node_modules/.bin/ember serve --no-build",
+    "fastboot-debug": "bash ./fastboot-debug.sh",
```

Comment:
you removed build from it, intentional?

Reply:
It was a mistake, thanks for the good catch. Re-added in https://github.com/Wikia/mobile-wiki/pull/618/commits/3b5f4f9786e0eb7b4d4d0544276ddea2302d8d20